From 8352c434e515f19dd1ba36256bfcc3ec9480838d Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Thu, 5 Mar 2015 22:38:52 +0100
Subject: [PATCH] imlib2 config

Call the imlib2-conf variable that are set by ac_cv_path_IMLIB2_CONFIG
with the absolute path because the imlib2-config file is not covered
by the PATH variable.
This is important for cross compiler that need to get the staging settings
instead of the host settings.

Signed-off-by: Carsten Schoenert <c.schoenert@gmail.com>
[ patch configure.in instead of configure to survive autoreconf ]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 configure.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index 38d7e78..fa5e87f 100644
--- a/configure.in
+++ b/configure.in
@@ -1094,8 +1094,8 @@ if test "x$enable_imlib2" = "xyes"; then
     AC_MSG_WARN([*** Imlib2 library not found, building without Imlib2 support ***])
     imlib2="no"
   else
-    IMLIB2_CFLAGS=`imlib2-config --cflags`
-    IMLIB2_LIBS=`imlib2-config --libs`
+    IMLIB2_CFLAGS=`$IMLIB2_CONFIG --cflags`
+    IMLIB2_LIBS=`$IMLIB2_CONFIG --libs`
     imlib2="yes"
   fi
 fi
-- 
2.1.4

